### PR TITLE
feat: [SRM-10250]: Add dynamic inputs to heathsource connector selection

### PR DESCRIPTION
**Pull request by vasuki-kalluri**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/85-cv/pages/health-source/HealthSourceDrawer/component/defineHealthSource/DefineHealthSource.tsx [hunk 1]
```diff
@@ -39,15 +42,16 @@ interface DefineHealthSourceProps {
 function DefineHealthSource(props: DefineHealthSourceProps): JSX.Element {
   const { onSubmit } = props
   const { getString } = useStrings()
+  const { expressions } = useVariablesExpression()
   const { onNext, sourceData } = useContext(SetupSourceTabsContext)
   const { orgIdentifier, projectIdentifier, accountId } = useParams<ProjectPathProps & { identifier: string }>()
   const { isEdit } = sourceData
+  const gitScope = useGitScope()
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/health-source/HealthSourceDrawer/component/defineHealthSource/DefineHealthSource.tsx:49]: we don't have git scope in CV , please remove

**Comment by deepesh-ui** [comment]
Please add screenshots

File: src/modules/85-cv/pages/health-source/HealthSourceDrawer/component/defineHealthSource/__tests__/DefineHealthSource.test.tsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ jest.mock('@cv/hooks/IndexedDBHook/IndexedDBHook', () => ({
 }))
 
 describe('DefineHealthSource', () => {
-  test('should have proper validation', async () => {
+  test.only('should have proper validation', async () => {
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/health-source/HealthSourceDrawer/component/defineHealthSource/__tests__/DefineHealthSource.test.tsx:54]: remove only

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 22 Issues 
* Code issue 22 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/8607)

**Pull request closed without merging**